manifests/charts/istio-control/istio-discovery/templates/istiod-injector-configmap.yaml

```diff
@@ -21,9 +21,11 @@ data:
   # to fine tune it or use it with kube-inject.
   config: |-
     # defaultTemplates defines the default template to use for pods that do not explicitly specify a template
-    # NOTE: this currently only allows a single element
     {{- if .Values.sidecarInjectorWebhook.defaultTemplates }}
-    defaultTemplates: {{ .Values.sidecarInjectorWebhook.defaultTemplates }}
+    defaultTemplates:
+{{- range .Values.sidecarInjectorWebhook.defaultTemplates}}
```

Comment:
I thought a list turned into a list?

```
$ ik manifest generate -d manifests --set .values.sidecarInjectorWebhook.defaultTemplates[0]=bar --force | grep defaultTemplates
        "defaultTemplates": [
    # defaultTemplates defines the default template to use for pods that do not explicitly specify a template
    defaultTemplates: [bar]
```

Seems to work ok, although I had to put `--force`

Reply:
If you specify two values, it becomes:
```
istioctl manifest generate --set .values.sidecarInjectorWebhook.defaultTemplates[0]=bar --set .values.sidecarInjectorWebhook.defaultTemplates[1]=foo --force | grep defaultTemplates
        "defaultTemplates": [
    # defaultTemplates defines the default template to use for pods that do not explicitly specify a template
    defaultTemplates: [bar foo]
```
and istiod unmarshals that to a single string (`bar foo`)..